### PR TITLE
test: fix mocked types

### PR DESCRIPTION
**Pull request by weichweich**
## Checklist:

- [x] I have verified that the code works
- [x] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [x] I have [left the code in a better state](https://deviq.com/boy-scout-rule/)
- [x] I have documented the changes (where applicable)

### CODE REVIEW

**Comment by flechtnergalani** [comment]
`any` chance you can get rid of the `any[]` in the signatures on `Verifier.ts` ?

File: src/actor/Verifier.spec.ts [hunk 1]
```diff
@@ -63,13 +63,12 @@ describe('Verifier', () => {
...
     blockchainApi.query.attestation.attestations.mockReturnValue(
-      new Option(
-        Tuple,
-        new Tuple(
-          [H256, AccountId, Text, Bool],
-          [cType.hash, attester.getAddress(), undefined, false]
-        )
-      )
+      new Option(Tuple.with([H256, AccountId, 'Option<H256>', Bool]), [
+        '0x1234', // ctype hash
```
Comment (LeonFLK) [inline src/actor/Verifier.spec.ts:67]: We should try to only use real hashes, and not these mocks, as this will be decoded to something unequal to 0x1234

**Comment by weichweich** [comment]
> `any` chance you can get rid of the `any[]` in the signatures on `Verifier.ts` ?

good catch! That's not that easy to solve. I bodged that a little bit for now. The gabi and public credentials are not the same. We should make them the same.  This is [tracked in an issue](KILTprotocol/ticket/issues/585)

File: src/actor/Verifier.spec.ts (hunk 1 above)
Comment (weichweich) [reply src/actor/Verifier.spec.ts]: I DID IT! 06e712f069524cbf1dd4ca1e8ebab7d63a4b996d

**Pull request merged**